pom.xml

```diff
@@ -21,7 +21,7 @@
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
-            <version>3.14.0</version>
+            <version>3.141.59</version>
```

Comment:
please move version to properties section
\+ same action for other dependencies

Reply:
moved all versions to properties section